crates/cxx-qt-lib/src/core/qmodelindex.rs

```diff
@@ -34,11 +34,21 @@ mod ffi {
         /// Returns the sibling at row for the current column. If there is no sibling at this position, an invalid QModelIndex is returned.
         #[rust_name = "sibling_at_row"]
         fn siblingAtRow(self: &QModelIndex, row: i32) -> QModelIndex;
+
+        /// Returns a `*mut c_void` pointer used by the model to associate the index with the internal data structure.
+        #[rust_name = "internal_pointer_mut"]
+        fn internalPointer(self: &QModelIndex) -> *mut c_void;
+
+        /// Returns a `*const c_void` pointer used by the model to associate the index with the
+        /// internal data structure.
+        #[rust_name = "internal_pointer"]
+        fn constInternalPointer(self: &QModelIndex) -> *const c_void;
```

Comment:
this only exists in Qt 6...

Reply:
Yeah, CI noticed as well :see_no_evil: 

So I guess we should really try getting the `#[cfg()]` attributes working in CXX then I guess.
Has there been any progress on this for #746 ?

Reply:
Not yet, we need to see if it works in a mini example then try and get it working with cxx-qt-lib

Reply:
I've removed the `const` version of this function for now.
It's not all that useful, as you can always cast to a `const*` from a `mut*` anyway.

We can always readd it if people ask for it.